Formula/synfig.rb

```diff
@@ -2,17 +2,11 @@ class Synfig < Formula
   desc "Command-line renderer"
   homepage "https://synfig.org/"
   license "GPL-3.0-or-later"
-  revision 2
 
   stable do
-    url "https://downloads.sourceforge.net/project/synfig/releases/1.4.2/synfig-1.4.2.tar.gz"
-    mirror "https://github.com/synfig/synfig/releases/download/v1.4.2/synfig-1.4.2.tar.gz"
-    sha256 "e66688b908ab2f05f87cc5a364f958a1351f101ccab3b3ade33a926453002f4e"
-
-    # Fix build with FFmpeg 5. Remove in the next release.
-    # Backport of upstream commit due to NULL -> nullptr changes.
-    # PR ref: https://github.com/synfig/synfig/pull/2734
-    patch :DATA
+    url "https://downloads.sourceforge.net/project/synfig/development/1.5.1/synfig-1.5.1.tar.gz"
+    mirror "https://github.com/synfig/synfig/releases/download/v1.5.1/synfig-1.5.1.tar.gz"
+    sha256 "aa91593c28a89f269be1be9c8bd9ecca6491f9e6af26744d1c160c6553ee2ced"
   end
```

Comment:
If we're removing the `patch` then the `stable` block can be removed. But I'd rather not switch this to `ffmpeg@4` especially when we have to add it to the allowlist to do so.

Reply:
Yeah I don't love using `ffmpeg@4` here.  I'm going to see if the patch still works, or if upstream has a commit to fix it.  Otherwise I'll get rid of the block.